plasmapy/formulary/tests/test_densities.py

```diff
@@ -0,0 +1,33 @@
+"""Tests for functionality contained in `plasmapy.formulary.frequencies`."""
+import astropy.units as u
+import numpy as np
+import pytest
+
+from plasmapy.formulary.densities import plasma_critical_density
+from plasmapy.formulary.frequencies import plasma_frequency
+
+n_i = 5e19 * u.m**-3
+ion = "e-"
+
+
+def test_plasma_critical_density():
+    r"""Test the plasma_critical_density function in frequencies.py."""
+
+    omega_p = plasma_frequency(n=n_i, particle=ion)
+    n_c = plasma_critical_density(omega_p)
+
+    assert n_c.unit.is_equivalent(u.m**-3)
+
+    assert np.isclose(n_c, 5e19 * u.m**-3)
```

Comment:
Note that this could also be written: 

```python
assert np.isclose(n_c.to(u.m**-3).value, 5e19)
```
which might be slightly better since it doesn't depend on how `np.isclose` decides to handle comparing `u. Quantity` objects?